README.md

```diff
@@ -75,12 +93,74 @@ The `onepassword.Item` model represents Items and `onepassword.Vault` represent
 The `connect.Client` also supports methods for:
 
 - listing Vaults
+```go
+vaults, err := client.GetVaults()
+if err != nil {
+	panic(err)
+}
+```
 - listing items in a Vault
+```go
+items, err := client.GetItems({vault_uuid})
+if err != nil {
+	panic(err)
+}
+```
 - searching by Item Title
+```go
+items, err := client.GetItemsByTitle({item_title}, {vault_uuid})
+if err != nil {
+	panic(err)
+}
+```
 - Retrieving Item by Vault and Item UUID
+```go
+item, err := client.GetItem({item_uuid}, {vault_uuid})
+if err != nil {
+	panic(err)
+}
+```
 - Creating Items in a Vault
+```go
+item := &onepassword.Item{
+	Fields: []*onepassword.ItemField{{
+		Value: "mysecret",
```

Comment:
Since the example Category is Login we should include the Username and Password purpose fields